ra2ce/analysis/indirect/losses.py

```diff
@@ -217,26 +217,23 @@ def _get_disrupted_criticality_analysis_results(
         else:
             criticality_analysis = criticality_analysis.drop_duplicates(["u", "v"])
 
-        if self.analysis.analysis == AnalysisIndirectEnum.SINGLE_LINK_LOSSES:
-            # filter out all links not affected by the hazard
-            if self.analysis.aggregate_wl == AggregateWlEnum.NONE:
-                self.criticality_analysis = criticality_analysis[
-                    criticality_analysis["EV1_ma"] != 0
-                ]
-            elif self.analysis.aggregate_wl == AggregateWlEnum.MAX:
-                self.criticality_analysis = criticality_analysis[
-                    criticality_analysis["EV1_max"] != 0
-                ]
-            elif self.analysis.aggregate_wl == AggregateWlEnum.MEAN:
-                self.criticality_analysis = criticality_analysis[
-                    criticality_analysis["EV1_mean"] != 0
-                ]
-            elif self.analysis.aggregate_wl == AggregateWlEnum.MIN:
-                self.criticality_analysis = criticality_analysis[
-                    criticality_analysis["EV1_min"] != 0
-                ]
-        else:
-            self.criticality_analysis = criticality_analysis
+        # filter out all links not affected by the hazard
+        if self.analysis.aggregate_wl == AggregateWlEnum.NONE:
+            self.criticality_analysis = criticality_analysis[
+                criticality_analysis["EV1_ma"] > float(self.analysis.threshold)
```

Comment:
`threshold` is already a `float`. I would not expect this casting to be needed. Please remove.